config/jobs/kubernetes-sigs/kubetest2/kubetest2-presubmits.yaml

```diff
@@ -0,0 +1,92 @@
+presubmits:
+  kubernetes-sigs/kubetest2:
+  - name: pull-kubetest2-verify-lint
+    decorate: true
+    always_run: true
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/krte:v20200726-f8d6253-master
+        command:
+        - wrapper.sh
+        args:
+        - make
+        - lint
+        securityContext:
+          privileged: true
+  - name: pull-kubetest2-verify-shellcheck
+    decorate: true
+    always_run: true
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/krte:v20200726-f8d6253-master
+        command:
+        - wrapper.sh
+        args:
+        - make
+        - shellcheck
+        securityContext:
+          privileged: true
+  - name: pull-kubetest2-build
+    decorate: true
+    always_run: true
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/krte:v20200726-f8d6253-master
+        command:
+        - wrapper.sh
+        args:
+        - make
+        - install
+        securityContext:
+          privileged: true
+  - name: pull-kubetest2-build-gce
+    decorate: true
+    run_if_changed: "kubetest2-gce/.*"
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/krte:v20200726-f8d6253-master
+        command:
+        - wrapper.sh
+        args:
+        - make
+        - install-deployer-gce
+        securityContext:
+          privileged: true
+  - name: pull-kubetest2-build-gke
```

Comment:
is there a reason to not have a single build job?
go caching should save some compute if we combine these.

Reply:
these will run only when the specific deployer code is touched. 
alternatively building all the deployers every time does add a non-trivial amount of time to each build.
unless we can add a smart build job that figures out the changed deployers, multiple job in parallel seems like the best approach (at the cost of having multiple jobs)?

Reply:
this is a bad idea, because the deployers also depend on code not in these paths, you will miss breaking changes from that direction.

a single build job will avoid re-building the same packages reapeatedly.
if it gets too slow we'll get persistent go caching

Reply:
unified into a single job. let' give that a go and see if we really need a go cache.